pypdf/_xobj_image_helpers.py

```diff
@@ -168,6 +168,24 @@ def _handle_flate(
     Process image encoded in flateEncode
     Returns img, image_format, extension, color inversion
     """
+
+    def bits2byte(data: bytes, size: Tuple[int, int], bits: int) -> bytes:
```

Comment:
Do we really needs this twice? https://github.com/py-pdf/pypdf/pull/2675/files#diff-a65edf4052008efeaf876fb2a61a82c587ecc844caebd573a4e536bfa13ff7adR126

Reply:
oups merging error